package.json

```diff
@@ -58,6 +58,7 @@
     "d3": "^7.8.2",
     "djv": "^2.1.4",
     "dompurify": "^2.4.0",
+    "electron": "23.0.0",
```

Comment:
I don't understand. Why we need electron?

Reply:
Oh! Never mind, this is just me being stupid and working on too many things at the same time BFC doesn't build on Electron. 🙈